lib/lighthouse_intent_to_file/monitor.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module LighthouseIntentToFile
+  class Monitor
```

Comment:
consider moving this to the `lighthouse/benefits_claims` as a subdirectory for intent_to_file?  could then in the future create a distinct intent_to_file service (and the other claims sub apis)

Reply:
Which one makes more sense? `lighthouse/benefits_claims/intent_to_file` or `lighthouse/intent_to_file`?

Reply:
While I would like to keep it completely separate it is a subapi of benefit claims and putting it there would keep it consistent with LH documentation